locales/es.po

```diff
@@ -108,64 +108,64 @@ msgstr ""
 #: src/renderer/pages/console_mirror/saved_connections_list/saved_connection_item.messages.ts:7:20
 #: src/renderer/pages/settings/help_page/support_box/network_diagnostics_button/network_diagnostic_result/network_diagnostics_result.messages.ts:24:20
 msgid "Available"
-msgstr ""
+msgstr "Disponible"
 
 #: src/renderer/pages/quick_start/stepper_dots/stepper_dots.messages.ts:3:15
 msgid "Back"
-msgstr ""
+msgstr "Atrás"
 
 #: src/renderer/pages/replays/replay_file_stats/game_profile_header/game_profile_header.messages.ts:2:24
 msgid "Back to replays"
-msgstr ""
+msgstr "Volver a las repeticiones"
 
 #: src/renderer/pages/settings/dolphin_settings/dolphin_settings.messages.ts:24:15
 msgid "Beta (Mainline)"
-msgstr ""
+msgstr "Beta (Principal)"
 
 #: src/renderer/pages/spectate/share_gameplay_block/broadcast_panel.messages.ts:9:50
 #. {0} broadcastDuration: string
 msgid "Broadcast ended after {0}"
-msgstr ""
+msgstr "Transmisión finalizada tras {0}"
 
 #: src/renderer/pages/spectate/share_gameplay_block/broadcast_panel.messages.ts:8:27
 msgid "Broadcast started"
-msgstr ""
+msgstr "Transmisión iniciada"
 
 #: src/renderer/pages/spectate/share_gameplay_block/start_broadcast_dialog.messages.ts:6:66
 #. {0} displayName: string
 #. {1} connectCode: string
 msgid "Broadcast to: {0} ({1})"
-msgstr ""
+msgstr "Transmitir a: {0} ({1})"
 
 #: src/renderer/pages/settings/help_page/support_box/network_diagnostics_button/network_diagnostic_result/network_diagnostics_result.messages.ts:30:27
 msgid "CGNAT or Double NAT"
-msgstr ""
+msgstr "CGNAT o NAT doble"
 
 #: src/renderer/components/activate_online_form/activate_online_form.messages.ts:5:23
 msgid "Can be changed later for a one-time payment"
-msgstr ""
+msgstr "Puede cambiarse después con un pago único"
```

Comment:
```suggestion
msgstr "Puede cambiarse después por un pago único"
```

- mistranslation: This is saying "It can be changed later with a unique payment." instead of **"for a"**...